### PR TITLE
nl.js update v4.2.2

### PR DESCRIPTION
**Pull request by mty22**
Submitted on behalf of nickauthentic

Closes: #219

### CODE REVIEW

File: nl.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ settings.interface.oldBrowser Gebruik Oude Browser
 settings.interface.rarityanim Schakel Zeldzaamheid Animaties uit
 settings.interface.showPfp Toon Profiel Avatar
 settings.interface.showTradeMsg Toon Trade Berichten
-settings.interface.hideNonTrade Hide Non-Tradable Skins
+settings.interface.hideNonTrade Verberg Non-Tradable Skins
```
Comment (jodiejomensen) [inline nl.js:179]: verberg niet verhandelbare skins

File: nl.js [hunk 2]
```diff
@@ -242,29 +242,29 @@ settings.crosshair.opacity.use Gebruik Eigen Ondoorzichtigheid
 settings.crosshair.opacity.custom Eigen Ondoorzichtigheid
 settings.crosshairm.header Crosshair (Throwing)
 settings.crosshairm.type Type
-settings.crosshairm.type.1 Shapes
-settings.crosshairm.type.2 Image
-settings.crosshairm.type.3 Precision
+settings.crosshairm.type.1 Vorm
```
Comment (jodiejomensen) [inline nl.js:245]: shapes = meervoud dus Vormen

File: nl.js [hunk 3]
```diff
@@ -242,29 +242,29 @@ settings.crosshair.opacity.use Gebruik Eigen Ondoorzichtigheid
...
+settings.crosshairm.style.0 Kruis
+settings.crosshairm.style.1 Holle Cirkel
+settings.crosshairm.style.2 Vaste cirkel
+settings.crosshairm.style.3 Holle Vierkant
+settings.crosshairm.style.4 Vaste Vierkant
+settings.crosshairm.image Afbeelding
+settings.crosshairm.image.paste Crosshair Afbeelding
+settings.crosshairm.image.width Afbeelding Breedte
+settings.crosshairm.image.height Afbeelding Hoogte
+settings.crosshairm.always Laat Altijd Zien
```
Comment (jodiejomensen) [inline nl.js:258]: Laat altijd zien ===> Altijd laten zien, laatste klinkt beter en mooier

File: nl.js [hunk 4]
```diff
@@ -289,12 +289,12 @@ settings.gameprivacy.header Gameplay Privacy
...
 settings.gameprivacy.anon Anonieme Modus
 settings.gameprivacy.hideGameTitle Verstop Game Titel
-settings.gameprivacy.verified Show Verified Badge
-settings.gameprivacy.premium Show Premium Badge
-settings.gameprivacy.partner Show Partner Badge
-settings.gameprivacy.rank Show Ranked Badge
+settings.gameprivacy.verified Verified Badge Weergeven
+settings.gameprivacy.premium Premium Badge Weergeven
+settings.gameprivacy.partner Partner Badge Weergeven
+settings.gameprivacy.rank Ranked Bagde Weergeven
```
Comment (jodiejomensen) [inline nl.js:295]: Kleine spellings fout kan gebeuren, "Badge weergeven" moet het zijn

File: nl.js [hunk 5]
```diff
@@ -823,7 +823,7 @@ windows.clan.leave Weet je zeker dat je de clan wilt verlaten?
 windows.clan.delete Weet je zeker dat je de clan wilt verwijderen?
 windows.clan.kick Weet je zeker dat je {0} wilt kicken van de clan?
 windows.store.premium.purchase Weet je zeker dat je {0} dagen Premium wilt kopen?
-windows.store.premium.gift Are you sure you want to Gift '{0}' {1} days of Premium?
+windows.store.premium.gift Weet Je Zeker Dat Je'{0}' {1} Dagen Van Premium Cadeur Wilt Doen?
```
Comment (jodiejomensen) [inline nl.js:826]: cadeau niet cadeur

**Review by jodiejomensen — CHANGES_REQUESTED** [review]

**Review by nickauthentic — APPROVED** [review]
Seems good.

**Comment by mty22** [comment]
hey @jodiejomensen -- does the latest changes look OK to merge? (submitted on behalf of @nickauthentic)

also please let me know what your Krunker IGN is

**Review by jodiejomensen — COMMENTED** [review]
> hey @jodiejomensen -- does the latest changes look OK to merge? (submitted on behalf of @nickauthentic)
> 
> also please let me know what your Krunker IGN is


I hope this is the right way to comment, the latest changes do look ok, my krunker ign is FlipBait (https://krunker.io/social.html?p=profile&q=FlipBait)

File: nl.js [hunk 6]
```diff
@@ -823,7 +823,7 @@ windows.clan.leave Weet je zeker dat je de clan wilt verlaten?
 windows.clan.delete Weet je zeker dat je de clan wilt verwijderen?
 windows.clan.kick Weet je zeker dat je {0} wilt kicken van de clan?
 windows.store.premium.purchase Weet je zeker dat je {0} dagen Premium wilt kopen?
-windows.store.premium.gift Weet Je Zeker Dat Je'{0}' {1} Dagen Van Premium Cadeur Wilt Doen?
+windows.store.premium.gift Weet Je Zeker Dat Je'{0}' {1} Dagen Van Premium Cadeu Wilt Doen?
```
[the remark this answers is not in the document]
Comment (nickauthentic) [reply nl.js:826]: Yeah my bad, changed it now!

**Pull request merged**